modules/ting/ting.entities.inc

```diff
@@ -366,6 +388,18 @@ class TingEntity extends DingEntity {
   public function getIsbn() {
     return empty($this->reply) ? FALSE : $this->reply->getIsbn();
   }
+
+  /**
+   * The Classification Text.
```

Comment:
Are you sure this should not go into https://github.com/easySuite/ding2/blob/development-4.4.x/modules/opensearch/src/OpenSearchTingObject.php#L161?

Reply:
Please relate to my comment.

Reply:
No. Because in a final we are processing TingEntity object which is going through TingEntity class declared in ting/ting.entities.inc.